src/py/flwr/server/rest_server/rest_api.py

```diff
@@ -61,7 +72,7 @@ def tasks(response: Response, client_id: Optional[int] = None) -> Response:
     )
 
 
-@app.post("/api/1.1/result/")
+@app.post("/api/1.1/result")
 async def result(request: Request) -> Response:  # Check if token is needed here
```

Comment:
The request can contain multiple results, should we make this plural?

```suggestion
async def results(request: Request) -> Response:  # Check if token is needed here
```